src/models/events.model.js

```diff
@@ -6,6 +6,30 @@ const EventStatus = {
   PROCESSED: 'Processed',
   FAILED: 'Failed',
 };
+
+const EVENT_TYPES = {
+  TRANSFER: 'Transfer',
+  GIVER_ADDED: 'GiverAdded',
+  PROJECT_ADDED: 'ProjectAdded',
+  CANCEL_PROJECT: 'CancelProject',
+  PROJECT_UPDATED: 'ProjectUpdated',
+  GIVER_UPDATED: 'GiverUpdated',
+  DELEGATE_ADDED: 'DelegateAdded',
+  DELEGATE_UPDATED: 'DelegateUpdated',
+  MILESTONE_COMPLETE_REQUEST_APPROVED: 'MilestoneCompleteRequestApproved',
+  MILESTONE_COMPLETE_REQUESTED: 'MilestoneCompleteRequested',
+  MILESTONE_COMPLETE_REQUEST_REJECTED: 'MilestoneCompleteRequestRejected',
+  PAYMENT_COLLECTED: 'PaymentCollected',
+  CONFIRM_PAYMENT: 'ConfirmPayment',
+  AUTHORIZE_PAYMENT: 'AuthorizePayment',
+  REQUEST_REVIEW: 'RequestReview',
+  SET_APP: 'SetApp',
+  NEW_APP_PROXY: 'NewAppProxy',
+  AUTO_PAY_SET: 'AutoPaySet',
+  APPROVE_COMPLETED: 'ApproveCompleted',
+  REJECT_COMPLETED: 'RejectCompleted',
+  RECIPIENT_CHANGED: 'RecipientChanged',
+};
```

Comment:
I saw there is no ducumentation for `event` field so I reach out to this enum, if you want to sure that there is no event missing, you can check this link out:

https://feathers.beta.giveth.io/events?event[$nin]=Transfer&event[$nin]=GiverAdded&event[$nin]=ProjectAdded&event[$nin]=CancelProject&event[$nin]=ProjectUpdated&event[$nin]=GiverUpdated&event[$nin]=DelegateAdded&event[$nin]=DelegateUpdated&event[$nin]=MilestoneCompleteRequestApproved&event[$nin]=MilestoneCompleteRequested&event[$nin]=MilestoneCompleteRequestRejected&event[$nin]=PaymentCollected&event[$nin]=ConfirmPayment&event[$nin]=AuthorizePayment&event[$nin]=RequestReview&event[$nin]=SetApp&event[$nin]=NewAppProxy&event[$nin]=AutoPaySet&event[$nin]=ApproveCompleted&event[$nin]=RejectCompleted&event[$nin]=RecipientChanged